akka-docs-dev/rst/stages-overview.rst

```diff
@@ -0,0 +1,134 @@
+Timer driven stages
+^^^^^^^^^^^^^^^^^^^
+
+*TODO*
+
+**It is currently not possible to build custom timer driven stages**
+
+.. _detached-stages-overview:
+Backpressure aware stages
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These stages are all expressible as a `DetachedStage`. These stages are aware of the backpressure provided by their
+downstreams and able to adapt their behavior to that signal.
+
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+Stage                  Emits when                                                                                                                  Backpressures when                                                                                                              Completes when
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+conflate               downstream stops backpressuring and there is a conflated element available                                                  never [2]_                                                                                                                      Upstream completes
+expand                 downstream stops backpressuring                                                                                             downstream backpressures                                                                                                        Upstream completes
+buffer (Backpressure)  downstream stops backpressuring and there is a pending element in the buffer                              buffer is full                                                                                                                                    Upstream completes and buffered elements has been drained
+buffer (DropX)         downstream stops backpressuring and there is a pending element in the buffer                                                never [2]_                                                                                                                      Upstream completes and buffered elements has been drained
+buffer (Fail)                                                                                                                                      fails the stream instead of backpressuring when buffer is full                                                                  Upstream completes and buffered elements has been drained
```

Comment:
“emits when downstream stops backpressuring and there is a pending element in the buffer”